src/api.js

```diff
@@ -423,7 +445,7 @@ class API {
     if (this.form) this.form.remove();
   }
 
-  _triggerCallback(callback, payload = {}) {
+  _triggerCallback(callback, payload, payload2, payload3) {
```

Comment:
payload2 and payload3 look odd. Can't we just put everything in `payload`?

Reply:
Got rid of this as you suggested.